features/interactive.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Interactive command
+
+  Scenario: Run command in interactive mode
+    When I run the command "DummyCommand" and I provide as input
+    """
+          1234 [enter]
+    """
+    Then the output should be
+    """
```

Comment:
Better way is to assert the real output from command. I think that we can make it more verbose for test purposes.